custom-words.txt

```diff
@@ -69,6 +69,7 @@ agri
 ahub
 ai
 aiapi
+aitoolchainoperator
```

Comment:
Please create a separate PR (to azure/main) to update custom-words.

Since the review cycle is very long and other people will also modify this file, it is very common to meet a merge conflict, and it is very troublesome to resolve the conflict for the dev branch.

Reply:
Removed and created a new PR to handle it [here](https://github.com/Azure/azure-rest-api-specs/pull/26136).